math/trigonometric.mbt

```diff
@@ -18,9 +18,12 @@ let pi4b = 0x0.000000A2216800000988
 
 let pi4c = 0x0.000000000000C234BE07
 
+pub const PI = 0x3.243F6A8885A308CA8A54
+
 pub let pi = 0x3.243F6A8885A308CA8A54
 
-let pi4 : Double = 4.0 / pi
+// todo: allow const here
+let pi4 : Double = 4.0 / PI
 
```

Comment:
we  need support this as well:
```
const Pi4 = 4.0 / PI
```

Reply:
It means we need to add a new keyword `const`?

Reply:
we already made const a keyword